src/bitmap.c

```diff
@@ -1510,8 +1510,8 @@ gdip_pixel_stream_get_next (StreamingState *state)
 			state->y++;
 
 			state->scan = (BYTE*)(state->data->scan0)
-			            + state->y * state->data->stride
-			            + state->x * gdip_get_pixel_format_bpp (state->data->pixel_format) / 8;
+						+ state->y * state->data->stride
+						+ state->x * gdip_get_pixel_format_bpp (state->data->pixel_format) / 8;
```

Comment:
and here. I'll stop commenting on these, I think we should just revert cases where the line is indented with tabs and spaces are just used for alignment.